Makefile

```diff
@@ -685,6 +685,13 @@ chkshsgr warn-shsgr tryshsgr.c compile load
 	hasshsgr.h
 	rm -f tryshsgr.o tryshsgr
 
+hasutmpx.h: \
+tryutmpx.c compile load
+	( ( ./compile tryutmpx.c && ./load tryutmpx ) >/dev/null \
+	2>&1 \
+	&& echo \#define HASUTMPX 1 || exit 0 ) > hasutmpx.h
```

Comment:
I would additionally add the ```#include <utmpx.h>``` here, so one does not need another ```#ifdef``` after including this file. At the end this would end up with 2 includes like it is done e.g. with fork.h.